### DISCUSSION TITLE
kong received the error 'failed to fetch SNI'

### DISCUSSION THREAD

**Discussion by zyiqian** [Kong/kong#12844] [Help]
Hi everyone..

When I deployed kong on k8s using dbless, we saw this type of error log intermittently.
```
failed to fetch SNI: failed to fetch '120.x.x.x' SNI: [off] must not be an IP, context: ssl_certificate_by_lua*, client: 162.x.x.x, server: 0.0.0.0:8443

```
I'm not sure if this error will affect the normal operation of our business. Can anyone help me? Thank you.

My KONG setup kong:3.1

**Comment by chronolaw** [comment]
It seems that your configuration has a incorrect sni value, could you check it again?

**Reply by zyiqian** [reply]
Could it be a malicious scan? I don't think it will affect the normal operation of the business.
